indico/modules/events/papers/controllers/management.py

```diff
@@ -51,3 +52,15 @@ def _process(self):
             logger.info("Paper managers of %r updated by %r", self.event_new, session.user)
             return jsonify_data(flash=False)
         return jsonify_form(form)
+
+
+class RHSwitchReviewingType(RHManagePapersBase):
+    """Enable/disable the paper reviewing types"""
+
+    def _process_PUT(self):
+        toggle_reviewing_type(self.event_new, request.view_args['reviewing_type'], True)
+        return jsonify_data(enabled=True)
```

Comment:
`flash=False` since you don't flash anything and reload (so even if you did flash, you wouldn't want it to be sent in the ajax response)